additional-resolver.ts

```diff
@@ -24,7 +24,7 @@ const resolvers: Resolvers = {
                   
                   if(!data || !data.companies) return [];
 
-                  return companiesIdsList.map( companyId => data.companies.find( company => company?.id === companyId))
+                  return data.companies
```

Comment:
you can leave this line as well but comnented `//return companiesIdsList.map( companyId => data.companies.find( company => company?.id === companyId))`